api/DashboardApi.ts

```diff
@@ -66,12 +66,12 @@ export class DashboardApi extends basem.ClientApiBase implements IDashboardApi {
 
             try {
                 let verData: vsom.ClientVersioningData = await this.vsoClient.getVersioningData(
-                    "5.0-preview.2",
+                    "5.1-preview.2",
                     "Dashboard",
                     "454b3e51-2e6e-48d4-ad81-978154089351",
                     routeValues);
 
-                let url: string = verData.requestUrl;
+                let url: string = verData.requestUrl!;

```

Comment:
The generation code was changed to add the ! at the end. Did all of the samples still run successfully?

Reply:
Yep! I think there's actually no way that could break things - the only thing it does is loosen how strict the typescript compiler is on null checks (it basically says that this thing can never be null so don't worry about that case).